### DISCUSSION TITLE
两种解码方式的区别

### DISCUSSION THREAD

**Discussion by 2607627233** [hooke007/mpv_PlayKit#178] [Q&A 问答]
HEVC和HEVC_CUVID有何不同
![QQ截图20220910175611](https://user-images.githubusercontent.com/43110396/189478706-e10c54f8-1b56-4f69-bcf8-b6c99a7c1145.png)
![QQ截图20220910175514](https://user-images.githubusercontent.com/43110396/189478708-9ee98f44-8d17-4e01-bb62-d6080e9097e6.png)

**Comment by hooke007** [comment]
cuvid是英伟达的解码器（好像是废话，图上写了

**Reply by hooke007** [reply]
顺带一提不知道为什么很多人都默认启用 `format[colormatrix=auto colorlevels=full]`
除了弄错几乎所有视频的色彩之外。。。我无法理解

**Reply by 2607627233** [reply]
我也是抄作业 看别人说有效就用了

**Reply by hooke007** [reply]
如何判定“有效”是否是正确行为？
几乎不存在需要使用这种滤镜的正规片源
(如果作为常驻选项而不是快捷键切换，更应该慎用“抄作业”)
